ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Packages.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTest, LedgerTestSuite}
+import io.grpc.Status
+
+final class Packages(session: LedgerSession) extends LedgerTestSuite(session) {
+
+  /** A package ID that is guaranteed to not be uploaded */
+  val unknownPackageId = " "
+
+  val listPackages =
+    LedgerTest("Packages", "Listing packages should return a result") { context =>
```

Comment:
The short test identifier should be different for each test (it's used to generate unique but recognizable party and command identifiers).